sdk/communication/communication-alpha-ids/review/communication-alpha-ids.api.md

```diff
@@ -0,0 +1,44 @@
+## API Report File for "@azure-tools/communication-alpha-ids"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference lib="esnext.asynciterable" />
+
+import { CommonClientOptions } from '@azure/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface AlphaIdConfiguration {
+    enabled: boolean;
+}
+
+// @public (undocumented)
+export class AlphaIdsClient {
+    constructor(connectionString: string, options?: AlphaIdsClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: AlphaIdsClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, options?: AlphaIdsClientOptions);
+    // (undocumented)
+    getConfiguration(options?: GetConfigurationOptions): Promise<AlphaIdConfiguration>;
+    // (undocumented)
+    upsertConfiguration(enabled: boolean, options?: UpsertConfigurationOptions): Promise<AlphaIdConfiguration>;
```

Comment:
what does enabled here refer to?

Reply:
So, this feature will be available for very specific customers, and even for those who get this, they will need to manually enable it. That's what the `upsertConfiguration` method does. `enabled` indicates whether Contoso want this feature for their resource.

I didn't document it because it's a private API. Do you believe I should leave a comment?